### ISSUE TITLE
Draft service blueprint

### ISSUE THREAD

**Issue by MelissaBraxton** [USDAForestService/USFS-timber-permitting#31]
**Background**
While we've drafted initial journey maps and task flows, we can now further flesh out our understanding of the as is system with a service blueprint. Service blueprints are particularly helpful tools for understanding and designing services that span multiple digital and physical artifacts as well as policies that we'll need to consider wrt to changes to any part of the process. 

**Acceptance criteria** 
- [x] accurate (walkthrough with product owner(s) reveals no discrepancies)
- [x] blueprint is accessible to both GSA and FS staff
- [x] Represents front/back of house tasks, touchpoints, systems and _policies_ relevant to each stage

   Service blueprint here: https://app.mural.co/t/gsa6/m/gsa6/1573664743449/005ff468f2c053f0ad4a86194889196cb178fab6

**Tasks**
- [x] Create initial draft blueprint in shareable format (Mural)
- [x] Research policies relevant to each stage
- [x] Cognitive walkthrough with PO

**Definition of done**
- [x]  PO approved

**Comment by cmajel** [comment]
Draft started here. Working walk through set for Monday.

https://app.mural.co/t/gsa6/m/gsa6/1573664743449/005ff468f2c053f0ad4a86194889196cb178fab6

**Comment by cmajel** [comment]
We had two awesome sessions this week with FS partners to walk through key parts of the service blueprint. 

Service blueprint here:
https://app.mural.co/t/gsa6/m/gsa6/1573664743449/005ff468f2c053f0ad4a86194889196cb178fab6

We've captured a lot and I feel like we've got placeholders for all the key stages of the process. There are definitely areas that continue to need investigation:
* The LEO in-field experience enforcing fuelwood permits
* Defining the right policy homes for certain actions (like load tag discretion and frontliner accountable property)
* Day of actions for permit holders, including how the actual cutting experience goes
* The actions taken after a permit is issued to connect up with FS financial systems

@MelissaBraxton do we need an additional PO walkthrough to close or can we count our two informational walkthroughs to this?
